### PR TITLE
From Acropolis To Rome

### PR DESCRIPTION
**Pull request by blrhc**
Newest blog post

### CODE REVIEW

File: blog/from-acropolis-to-rome/README.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+
+<p align="center"><img src="from-acropolis-to-rome.png"></p>
+
+#### Lead
+
+We bid farewell to Acropolis to welcome our new testnet, Rome!
+
+#### Body
+
+After almost nine months of experimentation by testnet participants on Acropolis, we are now ready to move on to our latest release, codenamed Rome, which incorporates a variety of new features.
```
Comment (bwhm) [inline blog/from-acropolis-to-rome/README.md]:
```suggestion
After a long development development cycle, we are ready to move on to our latest release, codenamed Rome, which incorporates a variety of new features.
```

File: blog/from-acropolis-to-rome/README.md [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+
+The Plan
+--------
+
+The main aspect of the release process for participants to be aware of is our intention to "kill" the Acropolis chain this Friday (13th March). The snapshot for Acropolis was taken at block number `4,718,300`.
+
+We will migrate all memberships and forum posts (but not media content) from Acropolis over to Rome and each existing membership will be given a small starting balance of `2000JOY`. If everything goes according to plan, the new chain for Rome will be launched on Monday (16th March) at approximately midday.
```
Comment (bwhm) [inline blog/from-acropolis-to-rome/README.md]:
```suggestion
We will migrate all memberships and forum posts made before the snapshot from Acropolis over to Rome and each existing membership will be given a small starting balance. If everything goes according to plan, the new chain for Rome will be launched on Monday (16th March) at approximately midday.
```

File: blog/from-acropolis-to-rome/README.md [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+
+#### Lead
+
+We bid farewell to Acropolis to welcome our new testnet, Rome!
+
+#### Body
+
+After almost nine months of experimentation by testnet participants on Acropolis, we are now ready to move on to our latest release, codenamed Rome, which incorporates a variety of new features.
+
+The Plan
```
Comment (bwhm) [inline blog/from-acropolis-to-rome/README.md]: This seems a little uncertain?

File: blog/from-acropolis-to-rome/README.md [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+--------
+
+The main aspect of the release process for participants to be aware of is our intention to "kill" the Acropolis chain this Friday (13th March). The snapshot for Acropolis was taken at block number `4,718,300`.
```
Comment (bwhm) [inline blog/from-acropolis-to-rome/README.md:30]: - Consider merging these three sections to two.
- Clarify that `Validators` and `Storage Providers` will receive a full weeks payment, but will not start with any privileges for `Rome.

File: blog/from-acropolis-to-rome/README.md [hunk 5]
```diff
@@ -0,0 +1,71 @@
...
+https://github.com/Joystream/joystream/issues/103
+
+What's New In Rome?
+-------------------
+
+Rome's main additions are the roles of `Content Curator` and `Content Creator`. We have also introduced the concept of channels, allowing one membership to create a selection of different channels on which to host content. Many other changes have been made to allow for easier navigation and to improve the user experience.
+
+We will be releasing information on Rome's incentive structure soon. This will provide rewards for validation, council membership and content storage (as before) but will also introduce a compensation structure for content creation and curation.
+
+After Rome
```
Comment (bwhm) [inline blog/from-acropolis-to-rome/README.md]: Skip this section?

File: blog/from-acropolis-to-rome/README.md [hunk 6]
```diff
@@ -0,0 +1,71 @@
...
+
+More information on the release timeline is available in the GitHub issue here:
+https://github.com/Joystream/joystream/issues/103
+
+What's New In Rome?
+-------------------
+
+Rome's main additions are the roles of `Content Curator` and `Content Creator`. We have also introduced the concept of channels, allowing one membership to create a selection of different channels on which to host content. Many other changes have been made to allow for easier navigation and to improve the user experience.
+
+We will be releasing information on Rome's incentive structure soon. This will provide rewards for validation, council membership and content storage (as before) but will also introduce a compensation structure for content creation and curation.
```
Comment (bwhm) [inline blog/from-acropolis-to-rome/README.md]: Soon -> Friday?

Also mention guides and binaries will be ready for release...

**Review by bwhm — CHANGES_REQUESTED** [review]
Looks good! Only minor changes suggested. Feel free to disregard them!

**Comment by blrhc** [comment]
Fixed issues highlighted.

**Review by bwhm — APPROVED** [review]
Looks good!

**Pull request merged**